src/test/java/staffconnect/model/person/ModuleTest.java

```diff
@@ -38,7 +38,7 @@ public void isValidModule() {
         assertFalse(Module.isValidModule("cs2103t")); // suffix not capitalised
         assertFalse(Module.isValidModule("cs2103T")); // prefix not capitalised
```

Comment:
Should we make module case-insensitive (but save as uppercase) to allow more flexibility?

Reply:
Sorry, I think this is under my job. I think I can implement that in the 1.2b, seperate from this PR :statue_of_liberty:

Reply:
Okay, no problem!